cli/console.ts

```diff
@@ -115,7 +116,7 @@ export function printCompiledGraph(graph: dataform.ICompiledGraph, verbose: bool
       writeStdOut(`${graph.tables.length} dataset(s):`);
       graph.tables.forEach(compiledTable => {
         writeStdOut(
-          `${datasetString(compiledTable.target, compiledTable.type, compiledTable.disabled)}`,
+          `${datasetString(compiledTable.target, tableEnumTypeToString(tableTypeFromProto(compiledTable, true)), compiledTable.disabled)}`,
```

Comment:
`compiledTable.type`?

Reply:
Changed to string representation of `compiledTable.enumType`.